cpp/data/rss_email.template.en

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+    <body>
+Dear ${user_name},
+
+The following items from your selected RSS feeds are new:</br>
+    <table id="table_delivered">
+        <thead>
+            <tr>
+                <th>Title</th>
+                <th>Description</th>
+                <th>URL</th>
+            </tr>
+        </thead>
+        <tbody>
+        {LOOP titles,descriptions,links}
+            <tr>
+                <td>{titles}</td>
+                <td>{descriptions}</td>
+                <td><a href="{links}"/></td>
```

Comment:
if the a has only a href and no content, you can neither see nor click it in the mail.

Reply:
Fixed.